.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,97 @@
+name: Release Docs
+
+on:
+  workflow_run:
+    workflows:
+      - Release
+    types:
+      - completed
+    branches:
+      - main
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: "release-docs"
+  cancel-in-progress: true
+
+jobs:
+  get_old_docs:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: get the gh-pages repo
+        uses: actions/checkout@v3
+        with:
+          ref: gh-pages
+
+      - name: tar the existing docs
+        run: |
+          mkdir -p ./docs/generated
+          tar -cvf old-docs.tar ./docs/generated
+
+      - name: create a document artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: old-docs
+          path: old-docs.tar
+
+  update_and_deploy_docs:
+    needs: get_old_docs
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout src
+        uses: actions/checkout@v3
+
+      - uses: ./.github/actions/git-config
+
+      - run: mkdir -p ./docs/generated
+      - name: Download the existing documents artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: old-docs
+
+      - run: tar -xf old-docs.tar ./docs/generated -C ./docs/generated
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
```

Comment:
Out of curiosity, why not the latest stable?

Reply:
No real reason, either would work.

Reply:
I'd vote for the stable then